base/common/src/main/java/com/netscape/certsrv/util/JSONSerializer.java

```diff
@@ -1,19 +1,51 @@
 package com.netscape.certsrv.util;
 
+import com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.module.jaxb.JaxbAnnotationIntrospector;
 
 /**
  * Implement this interface to provide default methods to serialize an object to/from JSON
  */
 public interface JSONSerializer {
 
+    default String toJSON(boolean requiresPrivateFieldAccess) throws Exception {
+        if (requiresPrivateFieldAccess) {
+            ObjectMapper mapper = new ObjectMapper();
+            return mapper.enable(SerializationFeature.INDENT_OUTPUT)
+                    .setVisibility(PropertyAccessor.FIELD, Visibility.ANY)
+                    .setAnnotationIntrospector(new JaxbAnnotationIntrospector(mapper.getTypeFactory()))
```

Comment:
Could we avoid using JAXB-related classes since we're going to drop them?

Reply:
Ah yes, I'll try `JacksonAnnotationIntrospector`.

Reply:
I ran a few tests with `toJSON()` using `JaxbAnnotationIntrospector` and `fromJSON()` using `JacksonAnnotationIntrospector`, and with both methods using `JacksonAnnotationIntrospector`, and all passed so I think this is fine.

Reply:
I couldn't find much documentation about `JacksonAnnotationIntrospector`. Does it work correctly if we don't specify it at all? I was thinking maybe it's the default introspector anyway, and maybe it's only needed to create custom introspectors.

Reply:
The `JaxbAnnotationIntrospector` was probably used since the class already has JAXB annotations. But since we're dropping JAXB, the those annotations needs to be replaced with Jackson annotations if necessary.

Reply:
By not specify it at all do you mean without `setAnnotationIntrospector`? The CI fails without this.

Reply:
Hmm.. I'm just curious the purpose of adding `JacksonAnnotationIntrospector`. I thought that would have been the default.

Regardless, the CI passed, so feel free to merge. Thanks!